cdsadapter/README.md

```diff
@@ -23,4 +23,4 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 Documentation
 =============
 
-Since CDS has been transferred to Eclipse OSGI, the Eclipse version cds is under package org.eclipse.osgi.internal.cds, with source code available [here](https://github.com/eclipse/rt.equinox.framework/tree/master/bundles/org.eclipse.osgi/container/src/org/eclipse/osgi/internal/cds).
+Since CDS has been transferred to Eclipse OSGI, the Eclipse version cds is under package org.eclipse.osgi.internal.cds, with source code available [here](https://github.com/eclipse-openj9/rt.equinox.framework/tree/master/bundles/org.eclipse.osgi/container/src/org/eclipse/osgi/internal/cds).
```

Comment:
This URL should not change.

Reply:
Updated. I'll work on fixing the other references in a different commit.